ExpressJS/utils/sequilize.js

```diff
@@ -0,0 +1,172 @@
+    { // Sequlieze connection for postgres
+      logging: console.log
+  });
+  console.log(`postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_HOST}:${process.env.POSTGRES_PORT}/waect`)
+
+} else if (dbConnectionType === 'prod') {
+
+  const ca_file = fs.readFileSync('/express-docker/secrets/ca-certificate.crt');
+
+  sequelize = new Sequelize(process.env.POSTGRES_NAME, process.env.POSTGRES_USER, process.env.POSTGRES_PASSWORD, {
+    host: process.env.POSTGRES_HOST,
+    port: process.env.POSTGRES_PORT,
+    dialect: 'postgres',
+    dialectOptions: {
+      ssl: {
+        ca: ca_file
+      }
+    }
+  });
+  //throw new Error('Not implemented yet');
+}
+
+//  -- Models --
+const Users = sequelize.define('Users', {
+  user_id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true // Indicates that it's a SERIAL column in PostgreSQL
+  },
+  username: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  pw_hash: {
+    type: DataTypes.STRING,
+    allowNull: false
+  }
+}, {
+  tableName: 'users', // Ensure Sequelize uses the correct table name
+  timestamps: false // Disable Sequelize's default timestamp fields
+});
+
+const Followers = sequelize.define('Followers', {
+  who_id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true
+  },
+  whom_id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true
+  }
+}, {
+  tableName: 'followers', // Ensure Sequelize uses the correct table name
+  timestamps: false // Disable Sequelize's default timestamp fields
+});
+
+const Messages = sequelize.define('Messages', {
+  message_id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true // Indicates that it's a SERIAL column in PostgreSQL
+  },
+  author_id: {
+    type: DataTypes.INTEGER,
+    allowNull: false
+  },
+  text: {
+    type: DataTypes.STRING,
+    allowNull: false
+  },
+  pub_date: {
+    type: DataTypes.INTEGER
+  },
+  flagged: {
+    type: DataTypes.INTEGER
+  }
+}, {
+  tableName: 'messages', // Ensure Sequelize uses the correct table name
+  timestamps: false // Disable Sequelize's default timestamp fields
+});
+
+// Define associations between models
+Messages.belongsTo(Users, { foreignKey: 'author_id' });
+
+// Define the many-to-many relationship between Users and Followers
+// Define associations
+// A user can follow many users
+Users.belongsToMany(Users, {
+  through: Followers,
+  as: 'UserFollowing',     // Users that this user is following
+  foreignKey: 'who_id',
+  otherKey: 'whom_id'
+});
+
+// A user can be followed by many users
+Users.belongsToMany(Users, {
+  through: Followers,
+  as: 'UserFollowers',     // Users that follow this user
+  foreignKey: 'whom_id',
+  otherKey: 'who_id'
+});
+// Test database operations
+async function testDatabase() {
```

Comment:
Is this function testing the schema?? If is the case we can delete the init_db from the db.js